safety_distance: 0.08

rgb_topic: /hsrb/head_rgbd_sensor/rgb/image_rect_color
depth_topic: /hsrb/head_rgbd_sensor/depth_registered/image_rect_raw # 16UC1 with depth in mm
cam_info_topic: /hsrb/head_rgbd_sensor/depth_registered/camera_info
point_cloud_topic: '/hsrb/head_rgbd_sensor/depth_registered/rectified_points'
global_frame: '/map'

grasping_pipeline:
  object_detector_topic: /object_detector/yolov5
  pose_estimator_topic: /pose_estimator/gdrnet
  result_visualization_service_name: '/pose_estimator/result_visualization_service'
  result_visualization_topic: /pose_estimator/result_visualization
  timeout_duration: 40.0

#pose_estimator:
  #unknown_object_grasp_detector_topic: /pose_estimator/find_grasppose_haf
  #detector_topic: /table_objects_extractor/get_label_image 
  # less relevant params